ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthSendRawTransaction.java

```diff
@@ -99,4 +114,26 @@ private Transaction decodeRawTransaction(final String hash)
       throw new InvalidJsonRpcRequestException("Invalid raw transaction hex", e);
     }
   }
+
+  private Transaction checkEIP1559(final Transaction transaction)
```

Comment:
I don't think we need any of this logic in the RPC stub. MainnetTransacitonValidator should be catching it when we try and `addLocalTransaction`.  The RPC stub should have no knowledge of the internals of the block chain.

Reply:
Ok done.